### PR TITLE
Replace the modal used to inform user that setting the certificate to NONE will remove any current certificates, with an inline warning message

### PR DESCRIPTION
**Pull request by kavinda1995**
### Related Issues
- https://github.com/wso2-enterprise/asgardeo-product/issues/10114

### Related PRs
- https://github.com/wso2/identity-apps/pull/3444

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:2]: Can we keep the year unchanged?
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx:2]: Can we keep the year unchanged?
```suggestion
 * Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx [hunk 3]
```diff
@@ -354,6 +367,30 @@ export const ApplicationCertificateWrapper: FunctionComponent<ApplicationWrapper
                                 readOnly={ readOnly }
                                 data-testid={ `${testId}-certificate-type-radio-group` }
                             />
+
+                            {
+                                showCertificateRemovalWarning && (
+                                    <Message
+                                        data-componentid="application-certificate-removal-warning-message"
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx:374]: Any reason to add both data-componentid and data-test id?

**Review by dasuni-30 — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx (hunk 3 above)
Comment (kavinda1995) [reply apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx:374]: Resolved in. https://github.com/wso2/identity-apps/pull/3462/commits/d131f1bd945f4ce8f681eb32dd7fb1261c0c5930

**Review by DonOmalVindula — APPROVED** [review]

**Pull request merged**